### DISCUSSION TITLE
Adding Concordence

### DISCUSSION THREAD

**Discussion by oparkins** [oparkins/Ayia#8] [Ideas]
We need to find an interlinear bible to use.

**Comment by depeele** [comment]
Several interlinear references I've found:
- https://ccat.sas.upenn.edu/gopher/text/religion/biblical/lxxmorph/
- https://biblehub.com/interlinear/
- https://www.biblestudytools.com/interlinear-bible/
- https://www.yumpu.com/en/document/view/63339526/free-download-read-the-interlinear-bible-hebrew-greek-english-english-hebrew-and-greek-edition-word
- https://berean.bible/downloads.htm
- https://www.scripture4all.org/

Related git projects:
- https://github.com/openscriptures
- https://github.com/tahmmee/interlinear_bibledata

Apps:
- https://www.reddit.com/r/Christianity/comments/nuo7ii/awesome_new_interlinear_bible_explorer_view/
- https://www.accordancebible.com/download-accordance/

Data formats:
- https://ubsicap.github.io/usfm/
- https://crosswire.org/osis/
